user/utils/utils.go

```diff
@@ -1,11 +1,28 @@
-package main
+package utils
 
 import (
 	"encoding/json"
 	"errors"
+	"os"
 	"strconv"
 )
 
+// GetConfig returns a configuration object from decoding the given configuration file
+func GetConfig(filePath string) (Config, error) {
```

Comment:
I’d personally have this return `(Config*, error)` so you can avoid returning an invalid config if there’s an error